### ISSUE TITLE
Can't send `Webauthn` between threads

### ISSUE THREAD

**Issue by drozdziak1** [kanidm/webauthn-rs#54]
I want to  use  a `Webauthn` struct as app data in an `actix-web` backend which needs to  share the struct between the worker threads of the tokio runtime. Whenever I try to use an `Arc<Mutex<Webauthn>>` as the type to put in `App::data()`, I get:
```plain
error[E0277]: `NonNull<rand::rngs::adapter::reseeding::ReseedingRng<rand_chacha::chacha::ChaCha20Core, rand_core::os::OsRng>>` cannot be sent between threads safely
  --> src/main.rs:67:5
   |
67 |     HttpServer::new(move || {
   |     ^^^^^^^^^^^^^^^ `NonNull<rand::rngs::adapter::reseeding::ReseedingRng<rand_chacha::chacha::ChaCha20Core, rand_core::os::OsRng>>` cannot be sent between threads safely
   |
   = help: within `Webauthn<config::Config>`, the trait `std::marker::Send` is not implemented for `NonNull<rand::rngs::adapter::reseeding::ReseedingRng<rand_chacha::chacha::ChaCha20Core, rand_core::os::OsRng>>`
   = note: required because it appears within the type `rand::rngs::thread::ThreadRng`
   = note: required because it appears within the type `Webauthn<config::Config>`
   = note: required because of the requirements on the impl of `std::marker::Send` for `futures::lock::Mutex<Webauthn<config::Config>>`
   = note: 1 redundant requirements hidden
   = note: required because of the requirements on the impl of `std::marker::Send` for `Arc<futures::lock::Mutex<Webauthn<config::Config>>>`
   = note: required because it appears within the type `[closure@src/main.rs:67:21: 130:6]`
   = note: required by `HttpServer::<F, I, S, B>::new`
```

**Comment by Firstyear** [comment]
Which version are you using in your cargo.toml please? I think this is fixed already in the 0.3 alpha that is currently still being worked on (there are some api-breaking changes coming).

**Comment by Firstyear** [comment]
As I believe this is fixed in 0.3, I'm going to close this,
